From 15596350b440730387028c759144e4b488c4fea1 Mon Sep 17 00:00:00 2001
From: Neal Liu <neal_liu@aspeedtech.com>
Date: Fri, 5 Sep 2025 19:34:56 +0800
Subject: [PATCH 16/32] configs: evb-ast2700: add ECDSA and ECDSA_VERIFY
 support

Add ECDSA and ECDSA_VERIFY configurations.

Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
Change-Id: I5d478c52c258ddf3f6047d8bd30b843eb48a7c4f
(cherry picked from commit 33d239c79b949633374b205fc59e3ec6857137c1)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 configs/evb-ast2700_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/evb-ast2700_defconfig b/configs/evb-ast2700_defconfig
index 205f719879..c8fca7c57d 100644
--- a/configs/evb-ast2700_defconfig
+++ b/configs/evb-ast2700_defconfig
@@ -164,6 +164,8 @@ CONFIG_ASPEED_UFS=y
 CONFIG_WDT=y
 # CONFIG_WDT_ASPEED is not set
 CONFIG_WDT_AST2600=y
+CONFIG_ECDSA=y
+CONFIG_ECDSA_VERIFY=y
 CONFIG_TPM=y
 CONFIG_LZ4=y
 CONFIG_LZMA=y
-- 
2.34.1

